packages/sources/wrapped/test/unit/adapter.test.ts

```diff
@@ -0,0 +1,31 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+
+  describe('validation error', () => {
+    const requests = [
+      { name: 'empty body', testData: {} },
+      { name: 'empty data', testData: { data: {} } },
+      {
+        name: 'symbol not supplied',
+        testData: { id: jobID, data: { symbol: '' } },
+      },
```

Comment:
Something to watch out for: before my comment above, this adapter would not require any input params, making this test incorrect (as we experienced with uscpi-one :wink:  )

Reply:
If we add the required field above, this test will be fine.